packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -221,11 +224,35 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
 
   /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
   const onInlineImageClicked = useCallback(
-    async (
-      attachment: FileMetadata,
-      onRenderTitleIcon?: (personaProps?: IPersonaProps) => JSX.Element
-    ): Promise<void> => {
-      const titleIcon = onRenderTitleIcon && onRenderTitleIcon();
+    async (attachmentId: string, messageId: string): Promise<void> => {
+      const messages = messageThreadProps.messages?.filter((message) => {
+        return message.messageId === messageId;
+      });
+      if (!messages || messages.length <= 0) {
+        console.log(`Message not found with messageId ${messageId}`);
```

Comment:
nitpick: we probably don't want to have these console.logs (similar for other console.logs in this PR)

Reply:
> nitpick: we probably don't want to have these console.logs (similar for other console.logs in this PR)

We currently don't have a logging system in Chat to only show error messages to Contoso developers.
I've seen MessageThread using console.log to display errors.
I'll bring this in the retro to introduce a proper logging system but I think it is out of the scope of this feature.